ert_shared/main.py

```diff
@@ -441,6 +441,15 @@ def start_ert_server(mode: str):
         yield
 
 
+def log_config(args: ArgumentParser, logger: logging.Logger) -> None:
+    if args.config is not None and os.path.isfile(args.config):
+        with open(args.config, "r", encoding="utf-8") as file_obj:
+            content = file_obj.read()
+            logger.debug(
```

Comment:
Depending on what you want to achive this should be `INFO` or above

Reply:
That is a good point. Motivation for this PR was a bug where arguments in the `FORWARD_MODEL` keyword are used in unexpected ways. We want to be able to gather information on that. Question is: is `DEBUG` good enough for that.

Reply:
@oyvindeide : When we log it as `DEBUG` will it end up in the azure logs? I think we would prefer `DEBUG` since it is really for us developers.

Reply:
> When we log it as DEBUG will it end up in the azure logs?

No, which was what I was getting at, but I now see that was very unclear from my comment, sorry about that. If that is the objective, it must be `INFO` or above.